tests/validators/array.test.ts

```diff
@@ -111,4 +119,13 @@ describe('ArrayValidator', () => {
 			});
 		});
 	});
+
+	test('GIVEN clone THEN returns similar instance', () => {
+		const arrayPredicate = s.string.array;
+		// @ts-expect-error Test clone
```

Comment:
Why is clone require-ing a ts-expect-error 👀

Reply:
It's a protected method